v2/internal/internalgauntlettest/gauntlet_test.go

```diff
@@ -99,6 +101,11 @@ func newChooser(t *testing.T, chooser string, dialer yarpc.Dialer, id yarpc.Iden
 		pl.Update(update)
 		return pl
 
+	case _tworandom:
+		pl := yarpctworandomchoices.New(dialer)
+		pl.Update(update)
```

Comment:
It would be neat to have more than one server as well as more than one client. Make the peer list work harder.

Reply:
🤔 I'd have to move some things around to get this working, I'll investigate this in a potential follow up